README.md

```diff
@@ -97,6 +97,44 @@ podman run --rm -p 8080:8080 \
   zot:latest
 ```
 
+# CLI
```

Comment:
Add a entry and link for this section at the top Summary like so:
command-line client support